### PR TITLE
docs: created documentation for capi yolov8 ensemble k8s

### PR DESCRIPTION
**Pull request by jim-wang-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
- [ ] No Employer Confidential details are added

## What requirement is this design document for?
detailed documentation for https://github.com/intel-retail/retail-use-cases/pull/41 

#**issue_number**

### CODE REVIEW

File: docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Yolov8 is one of the popular yolo object detection models. To show case the object detection of using YOLOv8, the efficientnet classification model will also be used to help validate the performance and accuracy of the C-API architecture.
+
+The yolov8 + efficientnet design will follow a similar custom pipeline as the previously implemented yolov5 + efficientnet profile. The same gstreamer decoding will be used for the input stream. C-API will send the frames to the OpenVINO Model Server(OVMS) processing through the custom pipeline defined in the config.json. OVMS will output the object bounding boxes along with the object classification. The profile will also output the processing latency in Frames Per Second (FPS).
+
+[![yolov8 + efficientnet profile-]](./images/yolov8-efficientnet-profile.jpg)
+
+## Applicable Repos
+
+[retail-use-cases-gst-capi-yolov8](https://github.com/intel-retail/retail-use-case/use-cases/gst_capi)
```
Comment (NeethuES-intel) [inline docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md:13]: getting 404 github error for this url

File: docs_src/use-cases/use-cases.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Intel Retail Use Cases
 
 [Automated Self Checkout](./automated-self-checkout/automated-self-checkout.md)
-[AI Connect for Scientific Data (AiCSD)](./AiCSD/aicsd.md)
\ No newline at end of file
+[AI Connect for Scientific Data (AiCSD)](./AiCSD/aicsd.md)
+[Retail Use Cases: C-API for YOLOV8 ensemble](./capi-yolov8-ensemble/capi-yolov8-ensemble.md)
```
Comment (NeethuES-intel) [inline docs_src/use-cases/use-cases.md:5]: can we make this as bullet points ? Currently all of them are displayed on one single line

**Review by NeethuES-intel — CHANGES_REQUESTED** [review]

File: docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md (hunk 1 above)
Comment (jim-wang-intel) [reply docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md]: fixed

File: docs_src/use-cases/use-cases.md (hunk 2 above)
Comment (jim-wang-intel) [reply docs_src/use-cases/use-cases.md]: fixed

**Review by NeethuES-intel — APPROVED** [review]
LGTM

File: docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Intel® C-API YOLOV8 Ensemble Object Detection Reference Package
+
+## Overview
+
+Yolov8 is one of the popular yolo object detection models. To show case the object detection of using YOLOv8, the efficientnet classification model will also be used to help validate the performance and accuracy of the C-API architecture.
```
Comment (antoniomtz) [inline docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md:5]:
```suggestion
YOLOv8 is one of the popular YOLO object detection models. To show case the object detection of using YOLOv8, the efficientnet classification model will also be used to help validate the performance and accuracy of the C-API architecture.
```

File: docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+The yolov8 + efficientnet design will follow a similar custom pipeline as the previously implemented yolov5 + efficientnet profile. The same gstreamer decoding will be used for the input stream. C-API will send the frames to the OpenVINO Model Server(OVMS) processing through the custom pipeline defined in the config.json. OVMS will output the object bounding boxes along with the object classification. The profile will also output the processing latency in Frames Per Second (FPS).
```
Comment (antoniomtz) [inline docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md:7]:
```suggestion
The YOLOv8 + efficientnet design will follow a similar custom pipeline as the previously implemented YOLOv5 + efficientnet profile. The same gstreamer decoding will be used for the input stream. C-API will send the frames to the OpenVINO Model Server(OVMS) processing through the custom pipeline defined in the config.json. OVMS will output the object bounding boxes along with the object classification. The profile will also output the processing latency in Frames Per Second (FPS).
```

**Review by antoniomtz — CHANGES_REQUESTED** [review]

**Review by brian-intel — APPROVED** [review]
LGTM

**Review by seanohair22 — CHANGES_REQUESTED** [review]

**Review by antoniomtz — APPROVED** [review]

File: docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Intel® C-API YOLOV8 Ensemble Object Detection Reference Package
+
+## Overview
+
+YOLOv8 is one of the popular YOLO object detection models. To show case the object detection of using YOLOv8, the efficientnet classification model will also be used to help validate the performance and accuracy of the C-API architecture.
+
+The YOLOv8 + efficientnet design will follow a similar custom pipeline as the previously implemented YOLOv5 + efficientnet profile. The same gstreamer decoding will be used for the input stream. C-API will send the frames to the OpenVINO Model Server(OVMS) processing through the custom pipeline defined in the config.json. OVMS will output the object bounding boxes along with the object classification. The profile will also output the processing latency in Frames Per Second (FPS).
+
+[![yolov8 + efficientnet profile-]](./images/yolov8-efficientnet-profile.jpg)
```
[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply docs_src/use-cases/capi-yolov8-ensemble/capi-yolov8-ensemble.md]: thanks, good catch, i fixed the syntax

File: docs_src/use-cases/capi-yolov8-ensemble/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    ```bash
+    cd ./use-cases/gst_capi/helm
+    ```
+
+and then build the container image for minikube to run:
+
+    ```bash
+    make minikube_build_capi_yolov8_ensemble
+    ```
+and you should see all built successfully.
```
[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply docs_src/use-cases/capi-yolov8-ensemble/getting_started.md]: all fixed, thank you

**Review by seanohair22 — APPROVED** [review]
LGTM!

**Pull request merged**